client/dive-common/components/ImportMultiCamDialog.vue

```diff
@@ -126,17 +126,26 @@ export default defineComponent({
     });
 
     async function open(dstype: DatasetType | 'calibration', folder: string | 'calibration') {
-      const ret = await openFromDisk(dstype);
+      const ret = await openFromDisk(dstype, dstype === 'image-sequence');
       if (!ret.canceled) {
         try {
           const path = ret.filePaths[0];
           if (folder === 'calibration') {
             calibrationFile.value = path;
+            if (ret.fileList?.length) {
+              emit('add-calibration-file', ret.fileList[0]);
+            }
```

Comment:
They are different things.  The `MultiCamImport(Keyword|Folder)Args` holds a string name for the calibration file.  This works on desktop.  The signals are emitting the HTMLFile reference used for uploading on the web.  I could modify both the MultiCamImportArgs to hold that data as well with optional parameters.  So either a combination with the multiCamArgs would have an additional parameter `htmlFileReferences`
```
interface HTMLFileReferences {
mediaHTMLFileList: Record<string, File[]>;
calibrationHTMLFile?: File;
}
```
Then this would be used to provide those file references required for upload when on the web..